gitea/release.go

```diff
@@ -46,6 +47,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAssets gets all the assets of a release in a repository
+func (c *Client) ListReleaseAssets(user, repo string, id int64) (*Release, error) {
```

Comment:
Ok, so everywhere in the code we still refer to them as Attachments, and the only place where "asset" appears is in the json for the release?